### PR TITLE
JBPM-6700: Upgrade to Lienzo 2.0.295-RELEASE & improved containment shape highlight

### PR DESCRIPTION
**Pull request by romartin**
Hey @manstis @tiagodolphine 

Here is the upgrade to latest lienzo release. It fixes a few bugs. This commit also improves the way the shapes are highlight (on containment etc). See [JBPM-6700](https://issues.jboss.org/browse/JBPM-6700) for more details.

**It depends on** https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/668

@tiagodolphine Assigning you as a reviewer and tester hehe, as @hasys is not feeling well. Lemme know whatever you need. I found a couple of bugs while testing this, but those are on lienzo side, so depends on a new release, in addition your work on bendpoints depends on this (you started on top of this version), so just let's move forward and I've created [this new ticket](https://issues.jboss.org/browse/JBPM-6735) to keep that bugs found present for next release.

Thanks!

### CODE REVIEW

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Comment by manstis** [comment]
Jenkins please retest this.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/main/java/org/kie/workbench/common/stunner/client/lienzo/shape/animation/ShapeViewDecoratorAnimation.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+    private List<com.ait.lienzo.client.core.shape.Shape<?>> getDecorators() {
+        return getSource().getDecorators();
+    }
+
+    com.ait.lienzo.client.core.animation.AnimationCallback getAnimationCallback() {
+        return animationCallback;
+    }
+
+    private final com.ait.lienzo.client.core.animation.AnimationCallback animationCallback =
```
[the remark this answers is not in the document]
Comment (romartin) [reply kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/main/java/org/kie/workbench/common/stunner/client/lienzo/shape/animation/ShapeViewDecoratorAnimation.java]: Hey @jomarko , cannot do that sorry, Stunner provides an AnimationCallback as well (by using another package), which is being used on the superclass, so here the FQCN is necessary.
BTW Thanks for testing!!

[the remark this answers is not in the document]
Comment (jomarko) [reply kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/main/java/org/kie/workbench/common/stunner/client/lienzo/shape/animation/ShapeViewDecoratorAnimation.java]: thanks for explanation and happy I helped with testing

**Comment by romartin** [comment]
Hey @tiagodolphine 
Thanks for testing it!! Good point, in fact the issue 2) should have been fixed by this PR, but I think I didn't tested that inside a parent (Lane), because on the canvas itself I fixed and was working... :( so good catch!
Lemme work on both points and will let you know ASAP!
Thanks!

**Comment by romartin** [comment]
Hey @tiagodolphine , had been working on your comments. About:
- 1) By looking around the code it's not easy as it seems.. IMO I would consider this minor priority as it requires a few days at least. So if you agree, I created a new [JBPM-6750](https://issues.jboss.org/browse/JBPM-6750) for it, and we can leave it for later... wDYT?
- 2) Yeah good catch. It's a bug on lienzo side, so unfortunately until next release it will be not fixed, but I have successfully reproduced and fixed it, so will be done once [6735](https://issues.jboss.org/browse/JBPM-6735) is closed.. This way, no code changes on Stunner side has been done.
Thanks!

**Pull request merged**